html/004c780c-30d4-461f-bccf-724358ff2f99.html

```diff
@@ -132,7 +132,7 @@ <h4>Try It: Simplify Expressions Using Properties of Exponents</h4>
   </ol>
 </div>
 <h4>Check Your Understanding</h4>
-<div class="os-raise-ib-pset" data-button-text="Check" data-content-id="d74162fd-64eb-4dfd-965f-ecaecd237636" data-fire-learning-opportunity-event="eventnameY" data-fire-success-event="eventnameX" data-retry-limit="3" data-schema-version="1.0"> ----------------------------------------
+<div class="os-raise-ib-pset" data-button-text="Check" data-content-id="d74162fd-64eb-4dfd-965f-ecaecd237636" data-fire-learning-opportunity-event="eventnameY" data-fire-success-event="eventnameX" data-retry-limit="0" data-schema-version="1.0"> ----------------------------------------
```

Comment:
I happened to :eye: these `----------------------------------------` which maybe someone may want to cleanup :man_shrugging:

Reply:
So many oddities...? I think we can make note for later...